samples/otlptraceexport/README.md

```diff
@@ -24,6 +25,7 @@ cd samples/otlptracexport && npm run start
 ```
 Alternate run scripts available:
```

Comment:
Any reason to give gRPC the shorter name of `start` instead of matching these names?

Reply:
Just that it was chosen be default. People are familiar with `npm run start` - so just wanted that command to work.